src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar calendar--start-day--mon calendar--month-length--29">
```

Comment:
class name should contain only one modifier
simply put, there could be only one `--`
```suggestion
    <div class="calendar calendar--start-day-mon calendar--month-length-29">
```